packages/cli/src/InternalHooks.ts

```diff
@@ -236,7 +236,9 @@ export class InternalHooks implements IInternalHooksClass {
 		data: IWorkflowExecutionDataProcess,
 	): Promise<void> {
 		void Promise.all([
-			Db.collections.Execution.update(executionId, { status: 'running' }),
+			Container.get(ExecutionRepository).updateExistingExecution(executionId, {
```

Comment:
same as above, could be injected into the constructor instead of getting it on every call

Reply:
Good point, this one worked well without issues. I'll try the same for the above class and update if possible.